pycode/memilio-epidata/memilio/epidata/modifyDataframeSeries.py

```diff
@@ -261,7 +268,58 @@ def insert_column_by_map(df, col_to_map, new_col_name, map):
     """
     df_new = df.copy()
     loc_new_col = df_new.columns.get_loc(col_to_map)+1
-    df_new.insert(loc=loc_new_col, column=new_col_name, value=df_new[col_to_map])
+    df_new.insert(loc=loc_new_col, column=new_col_name,
+                  value=df_new[col_to_map])
     for item in map:
         df_new.loc[df_new[col_to_map] == item[1], [new_col_name]] = item[0]
     return df_new
+
+
+def create_intervals_mapping(from_lower_bounds, to_lower_bounds):
```

Comment:
Could you additionally provide a more general function that inter/extrapolates age groups of different sizes/stratifications?

e.g. I start with a table

1-10 11-60 61-99
4    10     8

and have a population table (for simplicity let's assume it is equally distributed inside these age groups)

and I want to have 1-5 6-10 11-50 51-99?

This would then return (with the help of population data)

1-5 6-10 11-50 51-99
2   2    8    10

I hope I did the example correct :D

Reply:
Added a function` fit_age_group_intervals` in `modifyDataframeSeries.py` which fits your example. Also added some unit tests for this.